spec/src/main/asciidoc/microprofile-lra-spec.adoc

```diff
@@ -408,6 +408,8 @@ executing the `@LRA` annotated method, the implementation must be able to guaran
 participant has been or will eventually be registered with the LRA.
 This implies that the LRA is effectively validated as being active (since
 it is not possible to enlist with an inactive LRA).
+footnoteref:[note1,there is a caveat for nested LRAs which have an extra state model transition which allows an inactive LRA to transition from the `Closed` state to the `Cancelling` state. To facilitate this transition we allow enlistement if the implementation knows that the incoming context is nested and in the state `Closed` or `Active`. And in the case where the LRA is `Closed`, if the method finishes without error, the complete method, if present, would be invoked as normal.]
```

Comment:
Is the term `enlistment` correct in this context `To facilitate this transition we allow enlistement if the implementation knows that the incoming context is nested and in the state Closed`

The participant related to the nested LRA is already enlisted and instructed to complete. Later on, this same participant is called to compensate because of the cancel of the parent LRA.

For me, enlistment is when a participant is 'created' due to a user-initiated call where we need to make sure the compensate/complete is possible in the future.

Reply:
This caveat allows new participants to register with closed nested LRA's (provided that the parent LRA is still active).